### ISSUE TITLE
Collection object match task - more options to compare

### ISSUE THREAD

**Issue by AntWeb-org** [SpeciesFileGroup/taxonworks#3702]
### Feature or enhancement

When I select "compare" within the collection object match task, I would like to see more CO properties/Attributes available such as "Locatedat". Or better, have a customizable selection of fields to compare.

We note specimens as being on loan in the "Locatedat" field (see image) and once that field is updated for multiple specimens, I would like to be able to confirm any updates to that field via the "compare" tool.

### Location

Task - Collection Object Match

### Screenshot, napkin sketch of interface, or conceptual description

![image](https://github.com/SpeciesFileGroup/taxonworks/assets/74991451/906a1299-24f7-4486-a559-2374b45c3da0)

**Comment by AntWeb-org** [comment]
Forgot to add the team: @bpescador, @chrisgrinter, @rich-keller

**Comment by mjy** [comment]
Understood.  Does filter collection objects help in this regard at all?

**Comment by AntWeb-org** [comment]
Not really. The filter works to find COs in need of updating attributes, then I can export a list of those in need of updating.

In this case I already have a list of COs that I can copy and paste into the the CO Match for updating, and it would be more convenient to confirm all of the updates took place for each CO while I'm in the same task.
